back/app/models.py

```diff
@@ -62,4 +66,75 @@ def from_dict(cls, data):
             return cls(name=name, age=age, level=level, msg=msg, important=important, status=status, _id=_id)
         except Exception as e:
             raise ValueError(f"Error creating Data object from dict: {e}")
+        
+    @staticmethod
+    def get_statistics():
+        all_records = db.data.find({})
+        records_len = len(list(all_records.clone()))
+        avg_age=0
+        level_distribution={
+            "easy": 0,
+            "normal": 0,
+            "hard": 0
```

Comment:
wrong formating. "easy" : [numberOfEasy, precentageOfEasy]
so it becomes : 
level_distribution = {
    "easy" : [4,20],
    "normal" : [6,30],
    "hard" : [10,50]
}